.vib/suitecrm/goss/vars.yaml

```diff
@@ -0,0 +1,32 @@
+binaries:
+  - apachectl
+  - mariadb
+  - php
+  - render-template
+directories:
+  - mode: "0775"
+    paths:
```

Comment:
We should also add the `${SUITECRM_BASE_DIR}/tmp` directory to this list per the postunpack logic.

Reply:
OK! Added